chart/values.schema.json

```
@@ -100,6 +100,34 @@
                 }
             ]
         },
+        "securityContexts": {
+            "description": "Security context definition. The values in this parameter will be used when `securityContexts` is not defined for specific Pods/Container",
```

```suggestion
            "description": "Security context definition. The values in this parameter will be used when `securityContexts` is not defined for specific Pods/Container.",
```

Missing some punctuation.